src/ckb/ckbsettings.cpp

```diff
@@ -31,6 +59,42 @@ static QSettings* globalSettings(){
     return _globalSettings;
 }
 
+///
+/// \brief CkbSettings::_writable
+///
+bool CkbSettings::_writable = false;
+
+///
+/// \brief CkbSettings::isWritable Remember if the config files are writable (boolean getter)
+/// \return the value of CkbSettings::writable
+///
+bool CkbSettings::isWritable() { return _writable; }
+
+///
+/// \brief CkbSettings::setWritable Remember if the config files are writable (setter)
+/// \param v
+/// Set the value of CkbSettings::writable
+///
+void CkbSettings::setWritable(bool v) { _writable = v; }
+
+///
+/// \brief CkbSettings::checkIfWritable
+/// If the local implementation of the config database is not yet writable,
+/// bring up a popup to the user to informhim about it.
+/// Bring up the information where he can find the info.
+bool CkbSettings::informIfNotWritable() {
+    if (isWritable()) return false;
+    QMessageBox msgBox;
+    msgBox.setIcon(QMessageBox::Warning);
+    msgBox.setText("\n\n\nYour profile information for ckb-next is not writable.\n");
+    QString info = "This might happen, if you started the ckb-next program with root privileges earlier.\nOr did you copy it from somewhere?\n\nPlease have a look at "
```

Comment:
good function!
Now I can see my correction commit, but it is in the "normal" flow.
How can I mark  9556c84 as corresponding to your specific review?